src/Consolonia.Core/Helpers/Extensions.cs

```diff
@@ -12,5 +18,100 @@ public static void SubscribeAction<TValue>(
         {
             observable.Subscribe(new AnonymousObserver<AvaloniaPropertyChangedEventArgs<TValue>>(action));
         }
+
+        public static void Print(this IConsole console, PixelBufferCoordinate point, Pixel pixel)
+        {
+            console.Print(point,
+                pixel.Background.Color,
+                pixel.Foreground.Color,
+                pixel.Foreground.Style,
+                pixel.Foreground.Weight,
+                pixel.Foreground.TextDecorations,
+                pixel.Foreground.Symbol.Text);
+        }
+
+        /// <summary>
+        ///     Process text into collection of glyphs where a glyph is either text or a combination of chars which make up an
+        ///     emoji.
+        /// </summary>
+        /// <param name="text"></param>
+        /// <returns></returns>
+        public static IList<string> GetGlyphs(this string text)
```

Comment:
I think makes sense to declare as IReadOnlyCollection<string> or just string[] to indicate that returned vector can not be changed. Just a suggestion not to give impression, that returned object has some changable state